packages/react-components/src/components/VerticalGallery.tsx

```diff
@@ -114,8 +114,14 @@ export const VerticalGallery = (props: VerticalGalleryProps): JSX.Element => {
 
   return (
     <Stack className={mergeStyles(rootStyle, styles?.root)}>
-      <Stack className={mergeStyles(childrenContainerStyle, { '> *': styles?.children })}>
-        {childrenOnCurrentPage}
+      <Stack styles={{ root: childrenContainerStyle }}>
+        {childrenOnCurrentPage.map((child) => {
+          return (
+            <Stack.Item key={childrenOnCurrentPage.indexOf(child)} styles={{ root: styles?.children }}>
```

Comment:
nit: instead of `indexOf` in `key={childrenOnCurrentPage.indexOf(child)}`, `.map` gives you the index nicely:
```
{childrenOnCurrentPage.map((child, i) => {
   ....
    key={i}
```